recipes/auto-sklearn/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "auto-sklearn" %}
+{% set version = "0.12.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7741feee59a5dff12ae52cbc6df6c500bd094a76480a0a8f99ccd1c9d807aefd
+
+build:
+  number: 0
+  noarch: python
```

Comment:
Can you explain what is going on here? If `cython` is needed you probably need a compiler and it can't be `noarch: python`. If it isn't needed you probably don't need `numpy` in `host` and remove `pin_compatible` from `numpy` in `run`.

Reply:
Unfortunately not, as that is just a requirement from upstream: https://github.com/automl/auto-sklearn/blob/6e5886526ab01f97fbd842f6404badbe8a9aafad/setup.py#L23

I'm currently running a test without `numpy` and `cython` in host but @mfeurer is a maintainer upstream, do you know more about that?

Reply:
Cython is needed to compile the `ConfigSpace` package which is a dependency. The `ConfigSpace` package by now declares the build dependency on Cython itself. That means it should no longer be necessary to install Auto-sklearn itself. So the lines you point to can probably be deleted? I just started a test [here](https://github.com/automl/auto-sklearn/pull/1104).

Regarding the arch/noarch details I'm afraid I'm not familiar enough with conda-forge, but I hope my statement above clarifies the situation?

Reply:
The conda-forge version of `ConfigSpace` does list both dependencies so at least on conda-forge it shouldn't be required anymore then: https://github.com/conda-forge/configspace-feedstock/blob/8275d3e89cec704bad29fce60ee421c6d58af72c/recipe/meta.yaml#L22

Thanks for the tests, then I would suggest waiting until the new version is out since I cannot compile it due to failing dependencies, if I just remove`cython` and `numpy` from the recipe.

> Regarding the arch/noarch details I'm afraid I'm not familiar enough with conda-forge, but I hope my statement above clarifies the situation?

Yes it does, thanks!

Reply:
> Thanks for the tests, then I would suggest waiting until the new version is out since I cannot build it due to failing dependencies, if I just removecython and numpy from the recipe.

So should I just create a new release with this change?

Reply:
If that’s possible, that would be the best approach. I should also be able to patch this only for conda-forge but it’s always better to have it upstream.

Reply:
And here it is: https://pypi.org/project/auto-sklearn/0.12.5/

Reply:
Awesome, thanks! Recipe has been updated!